tests/src/CLRTest.Execute.Bash.targets

```diff
@@ -163,7 +163,7 @@ fi
         <Command><![CDATA[        export _DebuggerFullPath="${i#*=}"
         if [ ! -f "$_DebuggerFullPath" ]
         then
-            echo The Debugger FullPath \"$_DebuggerFullPath\" doesn\'t exist
+            echo "The Debugger FullPath "$_DebuggerFullPath" doesn't exist"
```

Comment:
@jasonwilliams200OK You're right, it does not need the single quote. I will fix it soon. But I want to review from maintainers first. This patch is been too long....